compose/cli/log_printer.py

```diff
@@ -13,11 +13,21 @@
 class LogPrinter(object):
     """Print logs from many containers to a single output stream."""
 
-    def __init__(self, containers, output=sys.stdout, monochrome=False, cascade_stop=False):
+    def __init__(self,
+                 containers,
+                 output=sys.stdout,
+                 monochrome=False,
+                 cascade_stop=False,
+                 follow=False,
+                 timestamps=False,
+                 tail="all"):
```

Comment:
Instead of passing these as individual properties, could we do it as a single dict `log_args` ?